android/src/main/java/com/it_nomads/fluttersecurestorage/FlutterSecureStoragePlugin.java

```diff
@@ -3,6 +3,8 @@
 import android.annotation.SuppressLint;
 import android.content.Context;
 import android.content.SharedPreferences;
+import android.os.Handler;
+import android.os.Looper;
 import android.util.Base64;
 import android.util.Log;
 import androidx.annotation.NonNull;
```

Comment:
Please remove androidx imports
It breaks non migrated apps

Reply:
Confused... This pull request didn't add androidx. Shouldn't it go away when merged?

Reply:
👍 you are right!